Geo-Cordinate Locator/script.py

```diff
@@ -0,0 +1,43 @@
+from tkinter import *
```

Comment:
Kindly add packages names in comment 
``` 
Package Included
1. pip install tkinter
2. pip install geopy
```

Or Better alternative use `requirements.txt` mentioning all the packages with version..